File: C99/1009.c
#include <stdio.h>
int main() {
	float salf, vendas;
	char nome[99];
	scanf(" %s", &nome);
	scanf("%f%f", &salf, &vendas);
	printf("TOTAL = R$ %.2f\n", salf + vendas * 0.15);
    return 0;
}

File: Javascript/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let nome = lines.shift();
let salf = parseFloat(lines.shift());
let comissao = parseFloat(lines.shift()) * 0.15;
console.log("TOTAL = R$", (salf + comissao).toFixed(2));
